packages/core/src/components/TimeAgo/TimeAgo.tsx

```diff
@@ -64,10 +64,9 @@ export const HvTimeAgo = ({
     showSeconds,
   });
 
-  // eslint-disable-next-line react/jsx-no-useless-fragment
-  if (justText && timestamp) return <>{timeAgo}</>;
-
-  return (
+  return justText && timestamp ? (
+    <>timestamp</>
```

Comment:
I believe it's `<>{timeAgo}</>` and not `<>timestamp</>`.

Reply:
Why not just `timestamp`, like it used to be? `string` is a valid `ReactNode`. Why convert it to a `ReactElement`? It's _kinda_ going against the `justText` prop too...

Reply:
I needed to cast it because of ts. let me know what you think.

Reply:
What's the TS issue? I couldn't reproduce it on my side... 🤔 
Do we need a `JSX.Element`/`EmotionJSX.Element` to always be returned?

If so, then the fragment is not so "useless" after all, at it might make sense to keep it...